vignettes/wrapping-apis.Rmd

```diff
@@ -799,20 +837,22 @@ httr2 will automatically refresh the token in two cases:
 
 -   The token includes an expiry date that's in the past.
 
--   The request returns a 401 status code and
+-   The request returns a 401 status code.
+
+JENNY COMMENT HERE
 
-If the access token has expired, httr will automatically retry automatically refresh before the first request.
+If the access token has expired, httr2 will automatically retry automatically refresh before the first request.
 If the request errors with a 401 and `invalid_token` error in the `WWW-authenticate` header.
 
 ### Caching
 
 By default, `req_oauth_auth_code()` and friends will cache the token in memory, so that multiple requests in the same session all use the same token.
 In some cases, you may want to save the token so that it's automatically used across sessions.
-This is easy to do (just set `cache_disk = TRUE` in `req_oauth_auth_code()`) but you need to carefully consider the consequences of saving the users credentials on disk.
+This is easy to do (just set `cache_disk = TRUE` in `req_oauth_auth_code()`) but you need to carefully consider the consequences of saving the user's credentials on disk.
 
 httr2 does the best it can to save these credentials securely.
 They are stored in a local cache directory (`rappdirs::user_cache_dir("httr2"))` that should only be accessible to the current user, and are encrypted so they will be hard for any package other than httr2 to read.
-However, there's no way to protect another R code from using httr2 to access them, so if you do choose to cache tokens, you should inform the user and give them the ability to opt-out.
+However, there's no way to protect other R code from using httr2 to access them, so if you do choose to cache tokens, you should inform the user and give them the ability to opt-out.
```

Comment:
```suggestion
However, there's no way to prevent other R code from using httr2 to access them, so if you do choose to cache tokens, you should inform the user and give them the ability to opt-out.
```

I saw this mis-wording once I was reviewing on GitHub.